Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/exchange/transform/fhir/FHIRTransformHelper.java

```diff
@@ -174,11 +174,14 @@ public static String extractEndpointName(String name) {
             try {
                 String serviceNames = PropertyAccessor.getInstance().getProperty(NhincConstants.FHIR_DIRECTORY_FILE,
                     nhinService.getUDDIServiceName());
-                if (isServiceNameMatch(serviceNames.split(SEPARATOR), name)) {
+                if (serviceNames == null) {
```

Comment:
Suggestion: use https://github.com/CONNECT-Solution/CONNECT/blob/9ad7f93b9448db68b0aabcf167d140adf80089fe/Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/properties/PropertyAccessor.java#L99
so that we don't need to catch exception in here.  Make code more readable